### PR TITLE
Lesson 1.4

### PR DESCRIPTION
**Pull request by cetinsuheyla**

### CODE REVIEW

File: src/TodoList.js [hunk 1]
```diff
@@ -1,34 +1,14 @@
...
-  return(
-    <ul>
-        {todoList.map((item) => {
-          return (
-            <TodoListItem key={item.id} item={item}/>
-          )
-        })}
-      </ul>
-  )
+function TodoList({ todoList }) {
```
Comment (davidrapoport) [inline src/TodoList.js:4]: another way you can destructure is to do this:

`function TodoList(props) {
    const {todoList} = props;
    .......
}`

Just FYI, your way is totally correct

File: src/TodoList.js [hunk 2]
```diff
@@ -1,34 +1,14 @@
...
+			{todoList.map((item) => {
+				return <TodoListItem key={item.id} item={item} />;
+			})}
+		</ul>
+	);
 }
 
-export default TodoList;
\ No newline at end of file
+export default TodoList;
```
Comment (davidrapoport) [inline src/TodoList.js:14]: did you install a new linter or some other extension? Just wondering

**Review by davidrapoport — APPROVED** [review]
Looks great!

**Comment by cetinsuheyla** [comment]
Hi David,

Thanks for your comments and for further info related to destructuring(That
is so valuable). What would be the reason for destructuring as you stated?
It seems like we are still using the props keyword. I am destructuring in
order to have cleaner code. Is there any specific situation that we want to
destructure in the way you showed?

I did not install any extension but my VS code style setting was not set to
the prettier extension, I changed it to prettier. I think that is the
reason it does not let me have a newline at the end of the file. I believe
I can change it.

Have a wonderful week,
Suheyla

**Comment by davidrapoport** [comment]
For your first point, I've seen it used when you have a lot of props and you are only using a small chunk of them in the current function. I agree that your way is cleaner right now when we don't have many props. Also your way is very helpful because it's self-documenting code.

**Comment by davidrapoport** [comment]
RE your second point, that makes sense. The code looks good after using `prettier`. What did you have to do to get it enabled in VSCode? I can't remember having to do anything on my end so I'm just wondering so I can pass along this information if anyone asks

**Pull request merged**